distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
now that shading is removed, we can remove this intermediate 'fully loaded'  as a followup: push its dependencies directly back into distribution/pom.xml, since all distributions will be including the optional dependencies.

This fully-loaded "dependency pom" indirection only was added to support the shaded jar.